sacred/observers/sql.py

```diff
@@ -17,14 +18,7 @@
 
 # ############################# Observer #################################### #
 
-class SqlObserver(RunObserver):
-    @classmethod
-    def create(cls, url, echo=False, priority=DEFAULT_SQL_PRIORITY):
-        engine = sa.create_engine(url, echo=echo)
-        session_factory = sessionmaker(bind=engine)
-        # make session thread-local to avoid problems with sqlite (see #275)
-        session = scoped_session(session_factory)
-        return cls(engine, session, priority)
+class SqlObserverBase(RunObserver):
```

Comment:
`Base` sounds a bit like abstract base class. I would probably call this something different, maybe `PlainSQLObserver`?